client/extensions/woocommerce/app/settings/payments/payment-method-edit-form-toggle.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FormToggle from 'components/forms/form-toggle';
+
+const PaymentMethodEditFormToggle = ( { checked, name, onChange } ) => {
```

Comment:
Since we will probably have other WooCommerce settings that return 'yes'/'no' that a helper component would be useful for, should we make this name a little more general and put it in `woocommerce/components`?